### DISCUSSION TITLE
Consider enabling MySQL strict mode

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5756] [Feature Requests]
<img width="1739" alt="Screenshot 2023-02-23 at 12 28 17" src="https://user-images.githubusercontent.com/40512816/224580669-a4e9cdc3-9760-45ab-83ca-a5eebaff5b72.png">

As suggested by the University of Florida

To do:
1. Research whether it's a good fit for us
2. Either provide it by default or provide instructions on how to enable it

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8219
